features/belongs_to.feature

```diff
@@ -21,9 +21,67 @@ Feature: Belongs To
     And I should see "Displaying 1 Post"
     And I should see a link to "Users" in the breadcrumb
     And I should see a link to "Jane Doe" in the breadcrumb
-    When I follow "Edit"
+
+  Scenario: Updating a child resource page
+    Given a configuration of:
+    """
+      ActiveAdmin.register User
+      ActiveAdmin.register Post do
+        belongs_to :user
+        permit_params :title, :body, :published_date if Rails::VERSION::MAJOR >= 4
+
+        form do |f|
+          f.inputs "Your Post" do
+            f.input :title
+            f.input :body
+          end
+          f.inputs "Publishing" do
+            f.input :published_date
+          end
+          f.actions
+        end
+      end
+    """
+    When I go to the last author's last post page
+    Then I follow "Edit Post"
```

Comment:
hehe, well I meant to change the following line, not this one. This seems part of the "cucumber setup", so `And` looked fine to me here.